mars/web/tests/test_api.py

```diff
@@ -204,6 +205,21 @@ def setUp(self):
 
         self.start_web(scheduler_address)
 
+    def tearDown(self):
+        self._web.stop()
+        self._pool.stop()
+
+    def start_web(self, scheduler_address):
+        import gevent.monkey
+        gevent.monkey.patch_all(thread=False)
+
+        web_port = str(get_next_port())
+        mars_web = MarsWeb(port=int(web_port), scheduler_ip=scheduler_address)
+        mars_web.start()
```

Comment:
`start` method supports `event`  parameter. I suggest to start the web by using the event, and do `event.wait(timeout=30` and then do timeout check in stead of keeping on loop which looks quite embarrassing.

Reply:
the same as the above comment.